.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: Continuous integration
+
+on: [pull_request]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: "yarn"
+
+      - name: Install
+        run: yarn
```

Comment:
Here is the place where I refered that I think that a similar behavior to` npm ci` would be something like `yarn install --immutable`. My mistake